controllers/articles.controller.js

```diff
@@ -2,10 +2,10 @@ const retrieveArticleById = require("../models/articles.model");
 
 function getArticleById(request, response, next) {
   const articleId = request.params.article_id;
-
+  console.log(articleId, "article ID");
   retrieveArticleById(articleId)
     .then((article) => {
-      response.status(200).send(article);
+      response.status(200).send({ article: article });
```

Comment:
nice update!.  Can also just send({article}).  Will give the object a key with the same variable name (article) as what's returned from then block, and that key will have value that matches the value stored in the article variable.  So sending {article} will look like {article: {article: object, with:properties, ..... }